pkg/target/regolib/src.rego

```diff
@@ -303,8 +303,12 @@ get_ns_name[out] {
   out := input.review.namespace
 }
 
+# JULIAN - This doesn't use the parameter...  
+# JULIAN - This just matches on all non-namespace objects that don't have a namespace specified.  I feel like it should be called "is_cluster_scoped"
 always_match_ns_selectors(match) {
+  # JULIAN - it's not a namespace
   not is_ns(input.review.kind)
+  # JULIAN - this will be true if namespace was blank or if it was not specified
   get_default(input.review, "namespace", "") == ""
```

Comment:
note, Rego has since added `object.get()` as a builtin, which is significantly more performant:

https://www.openpolicyagent.org/docs/latest/policy-reference/#objects-2

Reply:
There's a significant amount of code in `get_default` and `has_field`.  Rather than figure out if all the edge cases we covered manually are covered by `object.get()`, I'm going to add a TODO for the future.

Reply:
SGTM